src/specfem2d.cpp

```diff
@@ -95,7 +95,13 @@ void execute(const std::string &parameter_file, const std::string &default_file,
   //                    Read parameter file
   // --------------------------------------------------------------
   auto start_time = std::chrono::system_clock::now();
-  specfem::runtime_configuration::setup setup(parameter_file, default_file);
+  specfem::runtime_configuration::setup setup(parameter_file, default_file,
+#ifdef SPECFEMPP_BINDING_PYTHON
```

Comment:
This becomes tough to read IMO.

Reply:
This should be changed after we template YAML::Node